Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:18-alpine as builder
 ENV PNPM_HOME="/usr/bin"
 
 RUN apk add --no-cache git
-RUN npm install -g pnpm
+RUN npm install -g pnpm@7
```

Comment:
Should we be even more specific and install version 7.28.0, as that's the one set in `package.json`? (Or can we be less specific in `package.json`?)

Reply:
the package json one has no impact here. We probably want to upgrade to `pnpm@8` but that should be done independently.

Reply:
I guess I'm just saying there's an inconsistency where `package.json` says "you're supposed to install these dependencies with pnpm v7.28.0" (right?) and Dockerfile will actually install them with pnpm v7.30.5 because that's the latest v7.x.x. But maybe I'm missing the point of the `packageManager` key.

Reply:
I think you have a point @benface ! I am +1 for consistency with package.json